include/eve/detail/compress/simd/common/compress_using_masks.hpp

```diff
@@ -118,7 +118,7 @@ namespace eve::detail
   {
     if constexpr ( kumi::product_type<T> )
     {
-      return wide<T, N>{ kumi::map(compress_using_masks_shuffle, v, num) };
+      return wide<T, N>{ kumi::map([num](auto v_) { return compress_using_masks_shuffle(v_, num); }, v) };
```

Comment:
shouldn't it be a function object for inlining purpose ?

Reply:
Yeah it should. Let me do that in the next pr

Reply:
Perfect